settings.php

```diff
@@ -115,6 +115,7 @@
 
     $client = tool_objectfs_get_client($config);
     if ($client && $client->get_availability()) {
+        $PAGE->set_url('/admin/settings.php?section=tool_objectfs');
```

Comment:
This doesn't look right, the $PAGE->url->compare below is to detect when we are on this page, vs on the admin search page. If $PAGE is not set in the cli then instead you should only run define_client_check() below if $PAGE is set, and the url compare is good. That probably makes the session check above redundant too but it's safe to keep either way

Reply:
You're absolutely correct @brendanheywood 

Pushed a change.